hudi-common/src/test/java/org/apache/hudi/common/model/TestOverwriteNonDefaultsWithLatestAvroPayload.java

```diff
@@ -126,4 +127,33 @@ public void testDeletedRecord() throws IOException {
     assertEquals(payload1.combineAndGetUpdateValue(delRecord1, schema).get(), record2);
     assertFalse(payload2.combineAndGetUpdateValue(record1, schema).isPresent());
   }
+  @Test
```

Comment:
There is a checkstyle violation. Please add an empty line separator between methods.

Reply:
@codope done